lib/commands/update.js

```diff
@@ -227,6 +230,36 @@ class UpdateCommand extends Command {
         instance.version = version;
         instance.nodeVersion = process.versions.node;
     }
+
+    linkDefaultThemes({rollback}) {
+        const currentThemesDir = path.join(process.cwd(), 'current', 'content', 'themes');
+        const contentThemesDir = path.join(process.cwd(), 'content', 'themes');
+        // remove any broken symlinks caused by default themes no longer existing in previous version
+        if (rollback) {
+            if (fs.existsSync(currentThemesDir)) {
+                const installedThemes = fs.readdirSync(contentThemesDir);
```

Comment:
```suggestion
            if (fs.existsSync(contentThemesDir)) {
                const installedThemes = fs.readdirSync(contentThemesDir);
```

Not sure if this was intentional!

Reply:
TY! Fixed.